jsb/jsb-label.js

```diff
@@ -264,7 +264,7 @@ cc.Label = function (string, fontHandle, spriteFrame, fontSize) {
             customGlyphs: "",
             distanceFieldEnable: false
         };
-        label = jsbLabel.createWithTTF(ttfConfig, string, this._fontSize);
+        label = jsbLabel.createWithTTF(ttfConfig, string);
```

Comment:
If we pass the 3rd argument, it should be `hAlignment` rather than fontSize.
Font size is defined in ttfConfig.